node/chainSimulator/components/cryptoComponents.go

```diff
@@ -46,7 +46,7 @@ type cryptoComponentsHolder struct {
 }
 
 // CreateCryptoComponentsHolder will create a new instance of cryptoComponentsHolder
-func CreateCryptoComponentsHolder(args ArgsCryptoComponentsHolder) (factory.CryptoComponentsHolder, error) {
+func CreateCryptoComponentsHolder(args ArgsCryptoComponentsHolder) (factory.CryptoComponentsHandler, error) {
```

Comment:
Please rename to `CreateCryptoComponents`

Reply:
renamed